src/router/handlers/utils.ts

```diff
@@ -5,5 +5,31 @@
  */
 
 import { createHash } from 'crypto';
+import * as xss from 'xss';
 
-export const hash = (o: any): string => createHash('sha256').update(JSON.stringify(o)).digest('hex');
+const options = {
```

Comment:
Can we check with the security if this is the recommended library and setup to use? I remember an issue from another repo where a front end package was not sanitizing HTML as expected. Just want to make sure we're using the right package.

Reply:
Switched to the more reputable sanitize-html package